src/service/viewport-impl.js

```diff
@@ -430,8 +447,8 @@ export class Viewport {
     // TODO(erwinm): the duration should not be a constant and should
     // be done in steps for better transition experience when things
     // are closer vs farther.
-    return Animation.animate(this.ampdoc.getRootNode(), pos => {
-      this.binding_.setScrollTop(interpolate(pos));
+    return Animation.animate(this.ampdoc.getRootNode(), position => {
```

Comment:
update the comment above to point to https://github.com/ampproject/amphtml/issues/10463 (assign to me)